shop-the-look/app/globals.css

```diff
@@ -0,0 +1,70 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
There's a cleaner and native Next.js way to handle this for the future which prevents an additional network roundtrip for the font files: https://nextjs.org/docs/pages/building-your-application/optimizing/fonts